Cannot launch ng2-boostrap Demo

Hi, I try to launch ng2-bootstrap demo but not working.
I clone valor-software/ng2-bootstrap.
I write npm install & npm start and I have this error :

```
ERROR in Error encountered resolving symbol values statically. Could not resolve ng2-bootstrap/tabs relative to /home/axel/Developpement/Project/ng2-bootstrap/demo/src/app/api-docs/index.ts.,
resolving symbol NgApiDocModule in /home/axel/Developpement/Project/ng2-bootstrap/demo/src/app/api-docs/index.ts,
resolving symbol NgApiDocModule in /home/axel/Developpement/Project/ng2-bootstrap/demo/src/app/api-docs/index.ts

ERROR in ./demo/src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in '/home/axel/Developpement/Project/ng2-bootstrap/demo/src'
 @ ./demo/src/main.ts 4:0-74
 @ multi webpack-dev-server/client?http://0.0.0.0:4200/ ./demo/src/main.ts
webpack: Failed to compile.
```

Best Regards

--- a/package.json
+++ b/package.json
@@ -74,7 +74,7 @@
     "@angular/router": "^3.4.4",
     "@angular/tsc-wrapped": "0.5.1",
     "@ngtools/webpack": "1.2.7",
-    "@types/jasmine": "^2.5.41",
+    "@types/jasmine": "2.5.41",
     "@types/marked": "0.0.28",
     "@types/node": "7.0.0",
     "@types/webpack": "^2.2.2",

